### PR TITLE
[Merged by Bors] - feat: port Archive.Imo.Imo2005Q3

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by mo271** [comment]
I'm not sure about the removal of the aligns in namespace. Before the removal it had the error 
```
Declaration Imo2005Q3.key_insight not found.
```
and all the other imo files didn't have namespaces.

File: Archive/Imo/Imo2005Q3.lean [hunk 1]
```diff
@@ -24,8 +24,6 @@ and then making use of `xyz ≥ 1` to show `(x^5-x^2)/(x^3*(x^2+y^2+z^2)) ≥ (x
 -/
 
 
-namespace imo2005_q3
```
Comment (jcommelin) [inline Archive/Imo/Imo2005Q3.lean]: Why are you removing the namespace?

**Comment by jcommelin** [comment]
I think that recently all IMO files got namespaced.

File: Archive/Imo/Imo2005Q3.lean (hunk 1 above)
Comment (mo271) [reply Archive/Imo/Imo2005Q3.lean]: fixed: I put them back after I realized, I just have to change the name imo2005_q3 -> Imo2005Q3

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**